src/Core/Auth/UserFeatures/WebAuthnLogin/Implementations/CreateWebAuthnLoginCredentialCommand.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Bit.Core.Auth.Entities;
+using Bit.Core.Auth.Repositories;
+using Bit.Core.Entities;
+using Bit.Core.Utilities;
+using Fido2NetLib;
+
+namespace Bit.Core.Auth.UserFeatures.WebAuthnLogin.Implementations;
+
+internal class CreateWebAuthnLoginCredentialCommand : ICreateWebAuthnLoginCredentialCommand
+{
+    public const int MaxCredentialsPerUser = 5;
```

Comment:
❓ Is there a reason this is hard coded? might be worth plopping in a comment 🤔

Reply:
Just didn't really know where else to put it 🤷 Suggestions are welcome!

Reply:
Is the max number of credentials 5 for all security keys? or is this an arbitrary number we've selected? I don't really know enough of the fine details to suggest a comment. 🫠

Reply:
It's an arbitrary number picked by product to limit the amount of passkeys that users are allowed to register. It's an artificial limitation, i.e. from a purely technical perspective we could allow users to create hundreds